articles/extensions/custom-extensions.md

```diff
@@ -8,35 +8,109 @@ To begin creating an your custom extension, please feel free to fork/clone any o
 - [Auth0 Extension Boilerplate](https://github.com/auth0/auth0-extension-boilerplate)
 - [Auth0 Extension with API Boilerplate](https://github.com/auth0/auth0-extension-boilerplate-with-api)
 - [Auth0 Extension with React Boilerplate](https://github.com/auth0/auth0-extension-boilerplate-with-react)
+- [Auth0 Extension with Hooks](https://github.com/auth0/auth0-extension-boilerplate-hooks)
 
-Alternatively, you may following the Development Instructions provided via the "New Extension" window that appears when you click on the "+ Extension" button. This allows you to create your own extension using the command line.
+Alternatively, you may following the _Development Instructions_ provided via the _New Extension_ window that appears when you click on the _+ CREATE EXTENSION_ button. To view the _Development Instructions_, navigate to the [Extensions](${uiURL}/#/extensions) page of the [Management Portal](${uiURL}). Click on the _+ CREATE EXTENSION_ button. On the popup displayed click on the _Check out this command line tool_ link. The _Development Instructions_ popup is displayed. These instructions allow you to create your own extension using the command line.
+
+> Learn more about the `wt-cli` by visiting the [github repository](https://github.com/auth0/wt-cli) and the [documentation](https://webtask.io/docs/wt-cli).
 
 ## Installing a Custom Extension
```

Comment:
Also, we can add a line explaining that once it is installed you can use `wt update {extensionName} {file} -p {your-tenant-profile}`